lib/Migration/Version0Date20240826193657.php

```diff
@@ -194,8 +194,8 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
             $table->addColumn('version', Types::STRING, ['notnull' => true, 'length' => 255, 'default' => '0.0.1']);
             $table->addColumn('synchronization_id', Types::STRING, ['notnull' => true, 'length' => 255]);
             // Source
-            $table->addColumn('origin_id', Types::STRING, ['notnull' => false, 'length' => 255]);
```

Comment:
Is dit een revert of een fix, ik weet even niet meer 100% zeker of we nu getOriginId of getSourceId voor synchronizationContract in de synchronizationService gebruiken, wel belangrijk om dubbel te checken

Reply:
In de pr van dev naar main word dit aangegeven als een change, dus op main staat dit nu nog als source_id. Wat betekent dat alles in deze migration gelijk moet zijn en blijven met wat nu op main staat en alle changes in migration aangegeven in pr van dev naar main in de nieuwe migration moeten zitten
https://github.com/ConductionNL/openconnector/pull/52/files#diff-013049e2b4ee11df15d2a1adf145448f027dfc4625d7391ebafd0db66220ffa0